spec/support/twitter_timeline_tag_context.rb

```diff
@@ -0,0 +1,16 @@
+shared_context "with twitter_timeline_tag" do
```

Comment:
I'm not super keen on using shared contexts if there's nothing to share. I'd rather these things be extracted when they are needed to be extracted. The likeliness we need to check these expectations in multiple places are very slim (after all they are only supported in articles and comments). I'd rather this code stay in `spec/system/comments/user_fills_out_comment_spec.rb` for now

Reply:
Makes sense. Made changes.